backend/src/main/java/de/roman/toolio/service/InquiryPartService.java

```diff
@@ -0,0 +1,39 @@
+package de.roman.toolio.service;
+
+import de.roman.toolio.db.InquiryPartDb;
+import de.roman.toolio.model.InquiryPart;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+import java.util.List;
+import java.util.UUID;
+
+@Service
+public class InquiryPartService {
+
+    private final InquiryPartDb inquiryPartDb;
+
+    @Autowired
+    public InquiryPartService(InquiryPartDb inquiryPartDb) {
+        this.inquiryPartDb = inquiryPartDb;
+    }
+
+    public List<InquiryPart> listInquiryParts() {
+        return inquiryPartDb.findAll();
+    }
+
+    public InquiryPart addInquiry(InquiryPart inquiryPartToBeAdded) {
```

Comment:
Why do you set the id manually when mongodb will do this for you?

Please delete the comments

Reply:
This is done since I want to add the ID to the User that has created the inquiry. After the Item has been added to the MongoDB, the ID will be created by the DB and therefore I cannot access it properly with a unique identifier. Any suggestions? I will make sure to delete the comments in a future commit :) Thanks for the review!